### ISSUE TITLE
Feature: fullscreen and exit full screen animation

### ISSUE THREAD

**Issue by fastrizwaan** [solus-project/budgie-desktop#273]
Like we have for window appear/disappear (close), please enable fullscreen/exit fullscreen animations. gnome-shell has that, and it looks awesome.

**Comment by JoshStrobl** [comment]
> please enable fullscreen/exit fullscreen animations. gnome-shell has that, and it looks awesome

Yes, it has it and as a result, whenever you full-screen a video, like on Youtube, it will do a dumb fade out / fade in rather than just outright showing the video.

**Comment by ikeydoherty** [comment]
We might be able to do some special handling if we encounter apps we dont wanna see .. buggered up.

**Comment by fastrizwaan** [comment]
instead of fade in / out, appear disappear looks way better. e.g, launching an application and closing it.

**Comment by JoshStrobl** [comment]
> appear disappear looks way better

Yes...generally when you open an app, it appears. And when you close an app, it disappears.

**Comment by feskyde** [comment]
> appear disappear

Do you mean 'scaling' (windows grows from a point on the desktop to their normal size when opened and viceversa)?

**Comment by fastrizwaan** [comment]
Yes

**Comment by elbrook** [comment]
Personally, agree with Josh... Zoom-in to full screen (especially on videos) annoys me...
